README.md

```diff
@@ -9,98 +9,34 @@ While the below versions may work with the given versions of Kubernetes, Kubecos
 
 | Chart Version                  | Kubernetes Min | Kubernetes Max |
 |--------------------------------|----------------|----------------|
-| 1.107                          | 1.20           | 1.28           |
-| 1.108                          | 1.20           | 1.28           |
-| 2.1                            | 1.20           | 1.29           |
-| 2.2                            | 1.21           | 1.29           |
-| 2.3                            | 1.21           | 1.30           |
-| 2.4                            | 1.22           | 1.31           |
-| 2.5                            | 1.22           | 1.32           |
-| 2.6                            | 1.22           | 1.32           |
+| 2.7                            | 1.22           | 1.32           |
+| 2.8                            | 1.22           | 1.33           |
 
 ## Installation
 
-To install via Helm, run the following command.
+***Note: Upcoming changes to the Kubecost Helm chart will require changes.***
+
+To install the current GA version (2.8) of Kubecost via Helm, run the following command.
 
 ```sh
 helm upgrade --install kubecost -n kubecost --create-namespace \
-  --repo https://kubecost.github.io/kubecost/ kubecost \
-  --set kubecostToken="aGVsbUBrdWJlY29zdC5jb20=xm343yadf98"
+  --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer
 ```
 
 Alternatively, add the Helm repository first and scan for updates.
 
 ```sh
-helm repo add kubecost https://kubecost.github.io/kubecost/
+helm repo add kubecost https://kubecost.github.io/cost-analyzer/
 helm repo update
+helm install kubecost kubecost/cost-analyzer -n kubecost --create-namespace
 ```
 
-Next, install the chart.
-
-```sh
-helm install kubecost kubecost/kubecost -n kubecost --create-namespace \
-  --set kubecostToken="aGVsbUBrdWJlY29zdC5jb20=xm343yadf98"
-```
-
-While Helm is the [recommended install path](http://kubecost.com/install) for Kubecost, especially in production, Kubecost can alternatively be deployed with a single-file manifest using the following command. Keep in mind when choosing this method, Kubecost will be installed from a development branch and may include unreleased changes. We recommend using the manifest from a release branch, such as v1.108.
-
-```sh
-kubectl apply -f https://raw.githubusercontent.com/kubecost/kubecost-helm-chart/develop/kubecost.yaml
-```
-
-## Common Parameters
```

Comment:
We can fix this table for 3.0 with AI. Removing for now to avoid conflicting values.